site/styles/_icons.scss

```diff
@@ -31,3 +31,7 @@
   @extend .fa;
   @extend .fa-search;
 }
+
+main svg {
+  fill: $primary;
```

Comment:
This might have unintended consequences - we probably don't want to default every SVG in `main` to be primary.

Reply:
Not sure, but increases over time.